src/DBFTPlugin/ConsensusContext.cs

```diff
@@ -327,12 +329,37 @@ internal int GetExpectedBlockSizeWithoutTransactions(int expectedTransactions)
                 UInt256.Length +    // MerkleRoot
                 sizeof(ulong) +     // Timestamp
                 sizeof(uint) +      // Index
+                sizeof(uint) +      // Nonce
                 sizeof(byte) +      // PrimaryIndex
                 UInt160.Length +    // NextConsensus
                 1 + _witnessSize +  // Witness
                 IO.Helper.GetVarSize(expectedTransactions);
         }
 
+        private Tuple<byte[], uint> GetNonce(byte[] prikey)
+        {
+            byte[] aplha;
+            if (Block.Index > 1000)
+            {
+                // To prevent the primary uses the same block hash in Height (1000, 2000)
+                // also add the Height to the VRF input.
+                aplha = NativeContract.Ledger
```

Comment:
alpha?

Reply:
`alpha` is the term used by draft-irtf-cfrg-vrf-08 to name the vrf input. Check it https://tools.ietf.org/pdf/draft-irtf-cfrg-vrf-08

Reply:
I mean change `aplha -> alpha`

Reply:
lol，sorry  did not notice the typo.